lib/framework/action.dart

```diff
@@ -67,6 +67,18 @@ class ShowDialogAction extends EnsembleAction {
   final Map<String, dynamic>? options;
   final EnsembleAction? onDialogDismiss;
 
+  factory ShowDialogAction.from({Invokable? initiator, Map? payload}) {
```

Comment:
Why do we duplicate code? the fromYaml() below should work for regular Map also. Please rename it to `from` and test it with Map.